UIDokan/Sources/UIDokan/Style/EyeToggle.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  File.swift
+//  
+//
+//  Created by marwa on 13/07/2022.
+//
+
+import Foundation
+import UIKit
+
+
+let button = UIButton()
+let eyeButton = UIButton()
+
+extension UITextField {
+    
+    func eye(){
```

Comment:
What is eye? 😁
Methods are preferred to start with a verb like `show/add passwordToggle` or any other descriptive name.